libs/application/templates/rental-agreement/src/fields/PropertySearch/index.tsx

```
@@ -0,0 +1,498 @@
+                                                  }
+                                                  disabled={
+                                                    !checkedUnits[
+                                                      unit.unitCode ?? ''
+                                                    ]
+                                                  }
+                                                />
+                                              </T.Data>
+                                            </div>
+                                          </T.Data>
+                                        </tr>
+                                      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)